csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -4489,9 +4487,9 @@ The relevant path for this test is:
 
 ### 6.2.8 Use of MD5 as the only Hash Algorithm
 
-It must be tested that the hash algorithm `md5` is not the only one present.
+It MUST be tested that the hash algorithm `md5` is not the only one present.
 
-> Since collision attacks exist for MD5 such value should be accompanied by a second cryptographically stronger hash. This will allow users to double check the results.
+> Since collision attacks exist for MD5 such value SHOULD be accompanied by a second cryptographically stronger hash. This will allow users to double check the results.
```

Comment:
Same here: preferred rephrase to not have the shouting words, if not convincing keep lowercase, or if applicable augment the text part to normative.

Reply:
Addressed in [6356a59](https://github.com/oasis-tcs/csaf/pull/449/commits/6356a59f8647b9c1995408defb628b478783b622).